Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -702,8 +702,8 @@ void updateStereoBonds(RWMOL_SPTR product, const ROMol &reactant,
       Atom *pStart = pBond->getBeginAtom();
       Atom *pEnd = pBond->getEndAtom();
 
-      pStart->calcImplicitValence(true);
-      pEnd->calcImplicitValence(true);
+      pStart->calcImplicitValence(false);
+      pEnd->calcImplicitValence(false);
```

Comment:
These changes make sense to me.
@ricrogz : you wrote (I think ) the original version of this as part of #2553 
Do you see any reason to not make the change?

Reply:
@greglandrum, @mwojcikowski, I did indeed write that, but I cannot remember why I required strict valence calculation, nor can I think of a reason for it. I think it might even be a typo. I'm ok with this change.